### DISCUSSION TITLE
Updated Quire, back button in nav bar no longer working

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#198] [Get Help] [answered]
Hi all! Has anyone updated Quire to the latest version and had problems with the back button in the top navigation bar? I migrated two projects to the newest version of Quire, and now the back button on each page says "Tag" instead of the correct content title, and which I click the button, I get a "page not found" message. Please see two screenshots below. 

One of these projects previously had issues with the back button not displaying the correct content title (on some, but not all pages), but then the button still worked when clicked. 

<img width="1426" alt="Screen Shot 2021-10-07 at 10 04 43 AM" src="https://user-images.githubusercontent.com/68071544/136401253-eff43ba3-c923-4fa7-b905-299461ceb7d5.png">

<img width="1430" alt="Screen Shot 2021-10-07 at 10 06 40 AM" src="https://user-images.githubusercontent.com/68071544/136401281-10c4654f-f4ae-4504-8940-17114d1fbcbd.png">

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @zsofiaj I know restoring the theme resolved a number of your issues but I wanted to touch base. In particular, are you still encountering problems with the back button in your Quire project?

**Reply by zsofiaj** [reply]
Hi @Erin-Cecele Thanks so much for following up on this, and apologies for my slow reply! Yes, restoring the theme resolved a number of issues. I no longer have the problem noted above (back button is not linked and says "Tag" rather than the previous page title), but I do have a number of pages in the project that have a blank back button. See, for example, the screenshot below. The back button on these pages still works, but the title does not show. It does show on most pages. I can't figure out why it is missing on some. It seems random, but I'm sure there's an explanation!

<img width="1428" alt="Screen Shot 2021-12-01 at 9 48 23 AM" src="https://user-images.githubusercontent.com/68071544/144256166-dfd4f6a8-d959-4047-a97e-0b7f8d2ad969.png">
on

**Reply by geealbers** [reply]
Hi @zsofiaj, I think what's happening is that the titles are too long for the space in the navigation bar and this is making them disappear from view. This is something Quire could handle better (it would be better if they showed as cut-off or truncated rather than disappearing completely), but the solution is the same regardless, you can add a `short_title` to the page yaml for all the catalogue entries with long titles, as in the example below. And short titles should typically be no longer than around 30–40 characters. You can test them by previewing the publication in a browser and narrowing the window width. When they're right, the titles will only disappear when you get to something like a mobile device width, and in that case ALL the titles should disappear (as they're programmed to do for those smaller screen sizes) not just one of them.

```yaml
label: Cat. 8a–b
title: Master of Monteoliveto, *Virgin and Child Enthroned with Six Angels* and *The Crucifixion*
short_title: Master of Monteoliveto *Virgin* and *Crucifixion*
```

**Reply by zsofiaj** [reply]
Thanks so much, @geealbers. Yes, that fixed it! I didn't think length was an issue because some of the titles were appearing truncated on the buttons. See, for example, the two screenshots below (one has ellipses and the other doesn't). But I think those were all in the next button rather than the previous button. The titles that disappeared the previous button actually worked fine in the next button. I'm going to add short titles for all of the ones that were disappearing in the previous button. Or should I add short titles for all, even the ones that aren't causing a problem? I'm worried our curator won't love the short titles, but we shall see. 

<img width="1282" alt="Screen Shot 2021-12-01 at 2 09 18 PM" src="https://user-images.githubusercontent.com/68071544/144299517-1e3dbc65-2f84-44ad-8742-102b60da7dff.png">
<img width="1283" alt="Screen Shot 2021-12-01 at 2 10 00 PM" src="https://user-images.githubusercontent.com/68071544/144299550-14bde909-ea5b-4ed3-8320-39de6389fe40.png">

**Reply by zsofiaj** [reply]
I should also include this screenshot of one title that _does_ appear truncated in the previous button, in case it is helpful to know that some do work this way and some don't.
<img width="1287" alt="Screen Shot 2021-12-01 at 2 35 50 PM" src="https://user-images.githubusercontent.com/68071544/144301773-5b1ac85e-7564-4d61-b73a-4c57685225b9.png">
